test/integration/master_routes_test.go

```diff
@@ -33,25 +33,26 @@ var expectedIndex = []string{
 	"/apis/apps.openshift.io/v1",
 	"/apis/apps/v1beta1",
 	"/apis/authentication.k8s.io",
+	"/apis/authentication.k8s.io/v1",
 	"/apis/authentication.k8s.io/v1beta1",
 	"/apis/authorization.k8s.io",
+	"/apis/authorization.k8s.io/v1",
 	"/apis/authorization.k8s.io/v1beta1",
 	"/apis/authorization.openshift.io",
 	"/apis/authorization.openshift.io/v1",
 	"/apis/autoscaling",
 	"/apis/autoscaling/v1",
+	"/apis/autoscaling/v2alpha1",
```

Comment:
I don't know what our policy is on exposing alpha upstream apis. @smarterclayton?